Don't install translations. They get put directly in the Applications
folder where I doubt the app could find them.

Set the right path for the vnote.icns file in the Info.plist. The icns
file still doesn't get installed; since I don't know how to fix the
CMakeLists.txt to do that, the Portfile does it.
--- src/CMakeLists.txt.orig	2022-01-12 00:05:07.000000000 -0600
+++ src/CMakeLists.txt	2022-01-12 19:55:16.000000000 -0600
@@ -67,10 +67,9 @@
     endif()
     install(TARGETS VNote BUNDLE DESTINATION . COMPONENT Runtime
             RUNTIME DESTINATION bin COMPONENT Runtime)
-    install(FILES ${TRANSLATIONS} DESTINATION translations COMPONENT Runtime)
     set(MACOSX_BUNDLE_BUNDLE_NAME "VNote")
     set(MACOSX_BUNDLE_BUNDLE_GUI_IDENTIFIER "com.tamlok.VNote")
-    set(MACOSX_BUNDLE_ICON_FILE ${CMAKE_SOURCE_DIR}/src/resources/icons/vnote.icns)
+    set(MACOSX_BUNDLE_ICON_FILE "vnote.icns")
     set(MACOSX_BUNDLE_BUNDLE_VERSION "VNOTE ${PROJECT_VERSION_MAJOR}.${PROJECT_VERSION_MINOR}.${PROJECT_VERSION_PATCH}")
     set(MACOSX_BUNDLE_LONG_VERSION_STRING ${MACOSX_BUNDLE_BUNDLE_VERSION})
     # Set short version independent with project version to be able to increment independendently.
